src/docgen/transpile/python.ts

```diff
@@ -150,6 +150,7 @@ export class PythonTranspile extends transpile.TranspileBase {
       parentType: this.type(property.parentType),
       typeReference: typeRef,
       optional: property.optional,
+      variadic: false,
```

Comment:
This seems like a good reason to create a distinction between the `TranspiledProperty` and `TranspiledParam` interfaces. See https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys